ms2query/ms2library.py

```diff
@@ -200,12 +204,18 @@ def select_potential_true_matches(self,
                                         parent_masses_within_mass_tolerance]
             s2v_scores = self._get_s2v_scores(query_spectrum,
                                               selected_library_spectra)
-            found_matches = []
-            for i, selected_spectrum in enumerate(selected_library_spectra):
+            found_matches = pd.DataFrame(columns=["spectrum_id", "s2v_score"])
+            for i, spectrum_and_parent_mass in enumerate(parent_masses_within_mass_tolerance):
                 if s2v_scores[i] > s2v_score_threshold:
-                    found_matches.append(selected_spectrum)
-            found_matches_dict[query_spectrum_id] = found_matches
-        return found_matches_dict
+                    found_matches = \
+                        found_matches.append(
+                            {"spectrum_id": spectrum_and_parent_mass[0],
+                             "s2v_score": s2v_scores[i],
+                             "parent_mass_difference": spectrum_and_parent_mass[1]},
+                            ignore_index=True)
```

Comment:
```suggestion
                    found_matches.append(
                        {"spectrum_id": spectrum_and_parent_mass[0],
                         "s2v_score": s2v_scores[i],
                         "parent_mass_difference": spectrum_and_parent_mass[1]},
                        ignore_index=True)
```

Reply:
That should do I think...

Reply:
It is appending to a dataframe and not a list so I believe it has to be stored in the variable again